packages/twenty-server/src/engine/metadata-modules/serverless-function/dtos/function-parameter.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+
+import { IsString } from 'class-validator';
+
+@ObjectType()
+export class FunctionParameter {
+  @IsString()
+  @Field(() => String)
+  name: string;
+
+  @IsString()
+  @Field(() => String)
+  type: string;
```

Comment:
think we should not restrain to "simple" typing, we might want to provide objects FunctionParameter.type for example, like if we want a full record as a parameter.

Reply:
also, we should get the input schema for each serverless function version, I would store something like

```
{
"1": input_1,
"2": input_2,
...
}
```

Reply:
Checked with you. I will keep it like it is but renamed